oci/output.py

```diff
@@ -0,0 +1,26 @@
+import sys
+
+GREEN = "\033[1;32m"
+RED = "\033[1;31m"
+WHITE = "\033[1;37m"
+RESET = "\033[0m"
+
+def info(s, *args):
+    _output(s, *args)
+
+def failure(s, *args):
+    _output(s, *args, color=RED)
+
+def success(s, *args):
+    _output(s, *args, color=GREEN)
+
+def _output(fmt, args, color=None):
+    s = fmt % args
+    if sys.stdout.isatty():
+        s = _colorize(s, color)
+
+    sys.stdout.write(s + '\n')
+    sys.stdout.flush()
+
+def _colorize(s, color):
+    return (color or '') + s + RESET
```

Comment:
If we don't add color, why do need reset?

Reply:
Done